app/src/main/java/org/mozilla/fenix/components/toolbar/TabCounter.kt

```diff
@@ -217,14 +198,14 @@ open class TabCounter @JvmOverloads constructor(
     }
 
     companion object {
+        var INTERNAL_COUNT = 0
```

Comment:
Can this be an instance variable like it was before or is being static required? If it's required, then please make it internal.

Reply:
It's required because while navigating it got reset to 0.
Gonna make it internal.